app/views/application/_navbar.html.erb

```diff
@@ -1,85 +1,81 @@
-  <ul class="uk-navbar-nav uk-hidden-small">
-    <li aria-expanded="false" aria-haspopup="true" class="uk-parent uk-active" data-uk-dropdown="">
-      <a href=<%= profiles_me_path %> > <i class="uk-icon-user"></i> Profilom</a>
-      <div style="top: 40px; left: 0px;" class="uk-dropdown uk-dropdown-navbar uk-dropdown-bottom">
-        <ul class="uk-nav uk-nav-navbar">
-          <li><a href="#">Teljes profil</a></li>
-          <li><a href="#">Közösségi profil</a></li>
-          <li><a href="#">Közösségi történet</a></li>
-        </ul>
-      </div>
-    </li>
+<nav class="data-uk-sticky uk-navbar uk-hidden-small navbar">
+
+  <%= link_to "PékNext!", root_url, class:"uk-navbar-brand" %>
+  <ul class="uk-navbar-nav">
 
     <li aria-expanded="false" aria-haspopup="true" class="uk-parent" data-uk-dropdown="">
-      <a href=""><i class="uk-icon-institution"></i> Egységek</a>
+      <%= link_to "Profilom", root_url %>
       <div style="top: 40px; left: 0px;" class="uk-dropdown uk-dropdown-navbar uk-dropdown-bottom">
         <ul class="uk-nav uk-nav-navbar">
-          <li><a href="#">Körök</a></li>
-          <li><a href="#">Hierarchia</a></li>
+          <li><%= link_to "Szerkesztés", edit_profile_path(current_user.screen_name) %></li>
+          <li class="uk-nav-divider"></li>
+          <li><%= link_to "Kijelentkezés", controller: :login, action: :logout %></li>
         </ul>
       </div>
     </li>
+
     <li aria-expanded="false" aria-haspopup="true" class="uk-parent" data-uk-dropdown="">
-      <a href=""><i class="uk-icon-cog"></i> Fiók beállítások</a>
+      <a>Közösség</a>
       <div style="top: 40px; left: 0px;" class="uk-dropdown uk-dropdown-navbar uk-dropdown-bottom">
         <ul class="uk-nav uk-nav-navbar">
-          <li><%= link_to "Profil szerkesztése", edit_profile_path(current_user.screen_name) %></li>
-          <li><a href="#">SVIE tagság</a></li>
+          <li><%= link_to "Közösségi történet", root_url %></li>
+          <li><%= link_to "SVIE tagság", root_url %></li>
+          <li class="uk-nav-divider"></li>
+          <li><%= link_to "Körök", root_url %></li>
+          <li><%= link_to "Hierarchia", root_url %></li>
         </ul>
       </div>
     </li>
   </ul>
 
-  <div class="uk-nav-content uk-navbar-flip uk-hidden-small">
-
-    <%= form_tag(search_path, :method => 'get', :class => 'uk-form uk-margin-remove uk-display-inline-block') do %>
-      <p>
-        <%= text_field_tag :query, "", placeholder: "Név vagy kör", id: "search-field" %>
-        <%= button_tag "Search", :name => nil, class: 'uk-icon-search' do %>
-        <% end %>
-      </p>
+  <div class="uk-nav-content uk-navbar-flip search">
+    <%= form_tag(search_path, method: 'get', class: 'uk-form uk-margin-remove uk-display-inline-block') do %>
+      <%= text_field_tag :query, "", placeholder: "Név vagy kör", id: "search-field" %>
+      <%= button_tag "Search", name: nil, class: 'uk-icon-search' do %>
```

Comment:
Unfortunately `submit_tag` does not supported by UIkit icon, given in html class, so button_tag needed here.

I will consider `search` object, but there is another pull request preparing to search improvements, so it might be better done there.